### ISSUE TITLE
Add particle texture selector

### ISSUE THREAD

**Issue by SilverfishVR** [overte-org/overte#776]
I suggest adding a dropdown menu to the particle texture selector with a handful of default textures, it would cover most use cases and be way more convenient than having to find, host and link it yourself. The option to use a custom url should remain.

It would be very low overhead to host more particle textures on https://content.overte.org I think, they are very small, usually grayscale images.

Here's a few of my favorites: https://silverfish-freestuff.s3.amazonaws.com/index.html?prefix=Particles/

P.S. Something similar would be useful for the skybox textures, but they can be quite big so it may impact hosting cost.

**Comment by AleziaKurdis** [comment]
What would be the best for default particle... 
stored on the web or stored in resources ?

Originally it was in the resources, 
then Vircadia moved them on the web. 
then we bring them back to resource.

At this point. I just don't take any chances I store my own copy of the default with my project asset. 

And for Grey scale, it works now, but if we decide to support other mode. Like to support dark colors (to do dark smoke effect for example, as in SL). Then alpha would be necessary for the continuity. (this is no rocket science to convert grey scale to alpha anyway)

I definitely like the idea of a selection of default particles.

**Comment by SilverfishVR** [comment]
> What would be the best for default particle...
> stored on the web or stored in resources ?

I don't think it make much of a difference, but resources seems like the better option.

For skyboxes, having a few in resources would be nice, only cost is a few MB added to the install size.

Proper alpha cutout would be nice so you can have dark particles, but as I understand it it. may be a major change, and it would probably have some performance impact. Now, GPU particles, that is what is all the cool kids have 😄

**Comment by AleziaKurdis** [comment]
> may be a major change, and it would probably have some performance impact.

Seems to be a bitwise thing... Maybe @HifiExperiments could confirm.

**Comment by HifiExperiments** [comment]
re: black particles: currently we treat all particles as transparent, and transparent objects need to be sorted by distance in order to render correctly with normal blending.  since sorting all particles each frame is prohibitively expensive (although there are solutions for this), we instead render particles out of order with additive blending, which is why black doesn't show up (adding 0 does nothing).  opaque particles (just squares or with alpha cutout) are totally possible and would allow for black particles like you said.

in fact, I was thiiiiiiis close to adding them in 2019 😄: https://github.com/overte-org/overte/blame/00bcf0d41ff9d35b3f852a68df74c7eb27b7bc59/libraries/entities-renderer/src/RenderableParticleEffectEntityItem.cpp#L142

it should actually be super easy, I will take another look!

(and the rumors are that GPU particles are coming soon too)
